src/pages/store/index.tsx

```diff
@@ -1,71 +1,87 @@
-import { ItemCard, Navbar } from '@/components/store';
+import { Title } from '@/components/common';
+import { CollectionSlider, HelpModal, Hero, ItemCard, Navbar } from '@/components/store';
+import { config } from '@/lib';
+import { StoreAPI } from '@/lib/api';
 import withAccessType from '@/lib/hoc/withAccessType';
-import { PermissionService } from '@/lib/services';
-import { PrivateProfile } from '@/lib/types/apiResponses';
+import { CookieService, PermissionService } from '@/lib/services';
+import { PrivateProfile, PublicMerchCollection } from '@/lib/types/apiResponses';
+import { CookieType } from '@/lib/types/enums';
+import NoImage from '@/public/assets/graphics/cat404.png';
+import styles from '@/styles/pages/store/index.module.scss';
 import { GetServerSideProps } from 'next';
+import { useState } from 'react';
+
+type View = 'collections' | 'all-items';
 
 interface HomePageProps {
   user: PrivateProfile;
+  collections: PublicMerchCollection[];
 }
-const StoreHomePage = ({ user: { credits } }: HomePageProps) => {
-  // TEMP, just to demo the components
+const StoreHomePage = ({ user: { credits }, collections }: HomePageProps) => {
+  const [helpOpen, setHelpOpen] = useState(false);
+  const [view, setView] = useState<View>('collections');
```

Comment:
The state resets when you go to a collection page and go back, which is kind of annoying. I'm not sure if there's a way to store the state in the route instead